test/test_helper.rb

```diff
@@ -11,4 +11,12 @@
 
 require_relative 'helpers/rubocop_helper'
 
+# Uncomment this line to enable strict mode on the librray itself.
```

Comment:
```suggestion
# Uncomment this line to enable strict mode on the library itself.
```

You could use an environment variable.  That could also be used to run CI both with and without strict mode by providing the environment variable to one of them.

Reply:
Great idea